nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -738,10 +738,16 @@ public Description matchParameterizedType(ParameterizedTypeTree tree, VisitorSta
     if (!withinAnnotatedCode(state)) {
       return Description.NO_MATCH;
     }
+
     if (config.isJSpecifyMode()) {
-      GenericsChecks.checkInstantiationForParameterizedTypedTree(
-          tree, state, this, config, handler);
+      Symbol calledClass = ASTHelpers.getSymbol(tree);
```

Comment:
I think `baseClass` is a better name for this variable.  "called" implies something like a method call.